### PR TITLE
Issue #1504 Updating the Roadmap document

### PR DESCRIPTION
**Pull request by LalatenduMohanty**
Signed-off-by: Lalatendu Mohanty <lmohanty@redhat.com>

### CODE REVIEW

File: ROADMAP.adoc [hunk 1]
```diff
@@ -71,28 +71,41 @@ version name of a given release.
...
+[[minishift-180]]
+=== Minishift 1.8.0
+- [x] Add a minimum Openshift version for executing add-ons https://github.com/minishift/minishift/issues/1134[#1134]
 
-[[minishift-160]]
-=== Minishift 1.6.0
+[[minishift-190]]
+=== Minishift 1.9.0
+- [x] Provide notation for default value for addon-envs https://github.com/minishift/minishift/issues/1135[#1135]
+- [ ] Ability to use you own certificates for the cluster https://github.com/minishift/minishift/issues/391[#391]
```
Comment (hferentschik) [inline ROADMAP.adoc]: I don't think this is going to happen in this release. Even if started I have a hard time seeing it completed. IMO we might as well add it to 1.10.0

File: ROADMAP.adoc [hunk 2]
```diff
@@ -71,28 +71,41 @@ version name of a given release.
...
-=== Minishift 1.7.0
+[[minishift-1110]]
+=== Minishift 1.11.0
+- [ ] Add profile clone/copy command to reuse configs from existing profiles https://github.com/minishift/minishift/issues/1649[#1649]
 
+[[future]]
+=== Beyond Minishift 1.11.0
+- [ ] Use Minikube as dependency https://github.com/minishift/minishift/issues/1063[#1063]
+- [ ] Ability to manage users https://github.com/minishift/minishift/issues/390[#390]
+- [ ] Ability on running on existing Docker daemon https://github.com/minishift/minishift/issues/392[#392]
```
Comment (hferentschik) [inline ROADMAP.adoc]: I don't think this will ever be viable for us. But if you want to have it on the roadmap.

File: ROADMAP.adoc [hunk 3]
```diff
@@ -71,28 +71,41 @@ version name of a given release.
...
 - [ ] Ability interactively select OpenShift version https://github.com/minishift/minishift/issues/197[#197]
```
Comment (hferentschik) [inline ROADMAP.adoc]: Did we not say we get rid of this. There is so little value in this feature.

**Comment by LalatenduMohanty** [comment]
> Looking good. Feel free to merge. Personally, I'd move the certificate issue to 1.10. Up to you really.

I mostly agree, but as it is planned for this sprint I will leave it there. Also fixing the roadmap is a on-going work, so will fix it again in few weeks.

**Pull request merged**